jax/experimental/jax2tf/examples/saved_model_lib.py

```diff
@@ -77,49 +100,46 @@ def save_model(jax_fn: Callable,
       with_gradient=with_gradient,
       in_shapes=[None, shape_polymorphic_input_spec])
 
-  wrapper = _ExportWrapper(tf_fn, params, params_trainable=with_gradient)
+  # Create tf.Variables for the parameters.
+  param_vars = tf.nest.map_structure(
+      # If with_gradient=False, we mark the variables behind as non-trainable,
+      # to ensure that users of the SavedModel will not try to fine tune them.
+      lambda param: tf.Variable(param, trainable=with_gradient),
+      params)
```

Comment:
nit: the docstring specifies that this should be some kind of `tuple/list/dictionary`, but then do we need to use `tf.nest.map_structure`? Or maybe we want to abstract in the docstring to say that the params and inputs must be some kind of traversable?

Reply:
I do not think that I understand the comment. I think that the docstring is correct, and we do need to use `tf.nest.map_structure` to produce the `param_vars` with the same structure.